charts/super-agent-deployment/templates/deployment-superagent.yaml

```diff
@@ -67,6 +68,14 @@ spec:
             - name: super-agent-config
               mountPath: /etc/newrelic-super-agent
               readOnly: true
+            {{- range $subAgentName, $subAgentConfig := .Values.config.subAgents }}
+            {{- $chartSuffix := include "newrelic.common.naming.fullname" $ }}
```

Comment:
[just out of curiosity] why the variable is called "chartSuffix"?

Reply:
You are right. It should be `ReleasePrefix`. I don't know what I was thinking.

Reply:
Done on [#c7436bf](https://github.com/newrelic/helm-charts/pull/1221/commits/c7436bf5473024ce7a3034236bc97169695ae943)